example/case_study.jl

```diff
@@ -53,53 +61,52 @@ h_0(t, κ, λ) = κ/λ * (t/λ)^(κ - 1)
     # ---------------- Priors -----------------
     ## priors longitudinal
     # μ priors, population parameters
-    μ_BSLD ~ LogNormal(3.5,1)
-    μ_d ~ Beta(1,100)
-    μ_g ~ Beta(1,100)
-    μ_φ ~ Beta(2,4)
+    μ_BSLD ~ LogNormal(3.5, 1)
+    μ_d ~ Beta(1, 100)
+    μ_g ~ Beta(1, 100)
+    μ_φ ~ Beta(2, 4)
     μ = (BSLD = μ_BSLD, d = μ_d, g = μ_g, φ = μ_φ)
     # ω priors, mixed/individual effects
-    ω_BSLD ~ LogNormal(0,1)
-    ω_d ~ LogNormal(0,1)
-    ω_g ~ LogNormal(0,1)
-    ω_φ ~ LogNormal(0,1)
+    ω_BSLD ~ LogNormal(0, 1)
+    ω_d ~ LogNormal(0, 1)
+    ω_g ~ LogNormal(0, 1)
+    ω_φ ~ LogNormal(0, 1)
     Ω = (BSLD = ω_BSLD^2, d = ω_d^2, g = ω_g^2, φ = ω_φ^2)
     # multiplicative error
-    σ ~ LogNormal(0,1)
-    ## priors survival
-    κ ~ truncated(LogNormal(0,1),0.8,1.2)
-    λ ~ LogNormal(5,1)
-    
+    σ ~ LogNormal(0, 1)
+    ## prior survival
+    λ ~ LogNormal(5, 1)
+
     ## prior joint model
-    β ~ truncated(Normal(0,0.5),-0.1,0.1)
-    
+    γ ~ truncated(Normal(0, 0.5), -0.1, 0.1)
+
     ## η describing the mixed effects of the population
-    η_BSLD ~ filldist(Normal(0,Ω.BSLD),n)
-    η_d ~ filldist(Normal(0,Ω.d),n)
-    η_g ~ filldist(Normal(0,Ω.g),n)
-    η_φ ~ filldist(Normal(0,Ω.φ),n)
-    η = [(BSLD=η_BSLD[i], d=η_d[i], g=η_g[i], φ=η_φ[i]) for i in 1:n]
+    η_BSLD ~ filldist(Normal(0, Ω.BSLD), n)
+    η_d ~ filldist(Normal(0, Ω.d), n)
+    η_g ~ filldist(Normal(0, Ω.g), n)
+    η_φ ~ filldist(Normal(0, Ω.φ), n)
+    η = [(BSLD = η_BSLD[i], d = η_d[i], g = η_g[i], φ = η_φ[i]) for i = 1:n]
     # Transforms
-    Ψ = [[μ_BSLD * exp(η_BSLD[i]),
-    μ_g * exp(η_g[i]),
-    μ_d * exp(η_d[i]),
-    inverse(logit)(logit(μ_φ) + (η_φ[i]))]
-    for i in 1:n]
+    Ψ = [  [μ_BSLD * exp(η_BSLD[i]),
+            μ_g * exp(η_g[i]),
+            μ_d * exp(η_d[i]),
+            inverse(logit)(logit(μ_φ) + (η_φ[i])),] 
+         for i = 1:n]
 
     # ---------------- Likelihoods -----------------
     # add the likelihood of the longitudinal process
-    for measurement in 1:m
-        id = Int(longit_ids[measurement])
-        meas_time = longit_times[measurement]
+    for data in 1:m
```

Comment:
I think `data` is not a good variable name here for the iteration index, as it suggests that itself is already a data set or something like that. Recommend to either use an index like `j` or so that was used in the formulas, or something like `time_index`

Reply:
the main problem with single letter indexing is that we loop over some kind of "two dimensional" one-dimensional array (at least in the meaning). we have multiple observations for each individual individual, in my notation I use `j` for each . so maybe `ij` could be fitting, what do you think?

Reply:
typically that is just `j` then (similarly as you would write a double sum, first over `i` second over `j`. Main difference is that the range of `j` is depending on `i`

Reply:
in the paper i have the likelihood formulation:
![image](https://github.com/insightsengineering/JointSurvivalModels.jl/assets/5889127/8be0b9eb-80a1-46e7-90dd-f4e1ef64b0f4)
and the `data` here corresponds to the index `ij` and is used for both the time and observation arrays, which both have size `m`. So `ij` as an iterating variable for `1:m` somehow is natural in this case, though i agree that noramlly this could look different.

Reply:
ah ok got it, it is one long series of all observations. right, then `ij` sounds good.